src/test/java/com/amazon/opendistroforelasticsearch/sql/antlr/semantic/SemanticAnalyzerScalarFunctionTest.java

```diff
@@ -95,6 +96,7 @@ public void logFunctionCallWithUnknownFieldShouldPass() {
         validate("SELECT LOG(new_field) FROM semantics");
     }
 
+    @Ignore /** nested functions are blocked by throwing SqlFeatureNotImplementedException yet before implemented */
```

Comment:
I'm not sure I completely understand the explanation from the comment- are you saying that these tests throw the `SqlFeatureNotImplementedException` too quickly currently, which is why so many of them are set to `@Ignore`?

Reply:
@davidcui-amzn Yes, how shall I put it to make it clear?

Reply:
I think it is mostly clear- maybe just add something at the beginning such as `Test set to ignore due to nested functions blocked by...`

Reply:
Great, have changed them. Thanks!

Reply:
Great, have changed them. Thanks!